libpod/define/info.go

```diff
@@ -137,3 +144,9 @@ type Plugins struct {
 	// FIXME what should we do with Authorization, docker seems to return nothing by default
 	// Authorization []string `json:"authorization"`
 }
+
+type CPUUsage struct {
+	UserPercent   float64 `json:"user_percent"`
+	SystemPercent float64 `json:"system_percent"`
+	IdlePercent   float64 `json:"idle_percent"`
```

Comment:
hello, any reason to not use camelCase ?

Reply:
frankly, both docker and podman ue a mix of cases on json.  in this case, i think it makes sense to use camel.  Will fix.